core/src/test/java/apoc/text/StringsTest.java

```diff
@@ -388,11 +383,13 @@ public void testUrlDecode() {
 
     @Test
     public void testUrlDecodeFailure() {
-        thrown.expect(QueryExecutionException.class);
-        thrown.expectMessage("Failed to invoke function `apoc.text.urldecode`: Caused by: java.lang.IllegalArgumentException: URLDecoder: Incomplete trailing escape (%) pattern");
-        testCall(db,
-                "RETURN apoc.text.urldecode('ab+cd%3Dgh%26ij%3')  AS value",
-                row -> assertEquals("ab cd=gh&ij?", row.get("value"))
+        assertThrows(
+                "Failed to invoke function `apoc.text.urldecode`: Caused by: java.lang.IllegalArgumentException: URLDecoder: Incomplete trailing escape (%) pattern",
+                QueryExecutionException.class,
+                () -> testCall(db,
+                    "RETURN apoc.text.urldecode('ab+cd%3Dgh%26ij%3')  AS value",
+                    row -> assertEquals("ab cd=gh&ij?", row.get("value"))
```

Comment:
```suggestion
                    row -> {}
```

In case this one also throws, can not these assertions be removed from this test as well?